ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs

```diff
@@ -550,6 +550,17 @@ private void PopulateMonikers()
                             if (_frameworks.DocIdToFrameworkDict.ContainsKey(m.DocId))
                             {
                                 m.Monikers = new HashSet<string>(_frameworks.DocIdToFrameworkDict[m.DocId]);
+                                if (m.Signatures.IsProtected)
+                                {
+                                    //Filter out moniker of members that are in public sealed class;
+                                    m.Monikers = m.Monikers.Where(p =>
```

Comment:
this piece of logic seems duplicated with the `IsPublishSealedClass` property. Can we make it a shared method?

Reply:
it can't be shared, have different logic